build_tools/cmake/iree_bytecode_module.cmake

```
@@ -40,7 +41,7 @@ function(iree_bytecode_module)
   cmake_parse_arguments(
     _RULE
     "PUBLIC;TESTONLY"
-    "NAME;SRC;TRANSLATE_TOOL;CC_NAMESPACE"
+    "NAME;SRC;TRANSLATE_TOOL;CC_NAMESPACE;C_OUTPUT"
```

I think `C_OUPUT` is rather an `<options>` argument and not an `<one_value_keywords>` argument as suggested here, see https://cmake.org/cmake/help/latest/command/cmake_parse_arguments.html.

Suggested change together is the one above:
```suggestion
    "NAME;SRC;TRANSLATE_TOOL;CC_NAMESPACE"
```

---

Thanks for the suggestion. Also update the `build_tools/bazel_to_cmake/bazel_to_cmake_converter.py` accordingly.